src/compas/rpc/proxy.py

```diff
@@ -127,10 +122,10 @@ def start_server(self):
 
         """
         python = self._python
-        script = os.path.join(HERE, 'services', 'default.py')
+        args = [python, '-m', 'compas.rpc.services.default', str(self._port)]
```

Comment:
somehow these get printed to my editor, but i can't find a corresponding print statement...

Reply:
How is your setup? I tested from Grasshopper to CPython, but I haven't tried other modes. Is the example you commented below the one you use and prints this (from Rhino I assume)?

Reply:
yes, i tried in Rhino using the built-in editor.
didn't always print this, though...